README.md

```diff
@@ -28,15 +37,48 @@ import yt_idefix
 
 Now `yt.load` will be able to read Pluto/Idefix output files.
 
-## Supported formats
+### Dataset's Metadata
+The metadata is automaticly parsed from data file, definitions header file and inifile when loading dataset.
 
-| Code   | format | supported since | additional dependencies |
-|--------|:------:|:---------------:|-------------------------|
-| Idefix | `.dmp` | v0.1.0          |                         |
-| Idefix | `.vtk` | v0.3.0          |                         |
-| Pluto  | `.vtk` | v0.9.0          |                         |
-| Pluto  |  XDMF  | v1.1.0          | `h5py`                  |
+definitions header file (`definitions.h` for Pluto, or `definitions.hpp` for Idefix) and inifile (`.ini` file) are assumed to be along with data file. Their paths can also be specified explicitly with paramerters `definitions_header` and `inifile`.
 
+Besides, the geometry of dataset can be given by `geometry` parameter ("cartesian", "spherical", "cylindrical" and "polar")  when it cannot be parsed from file.
+
+```
+# Examples
+ds = yt.load("data.0010.vtk", definitions_header="../definitions.h", inifile="example.ini")
+ds = yt.load("data.0010.vtk", geometry='spherical")
+```
+### Unit System
+The data are loaded as physical quantities with units. The default unit system is `cgs` in yt. This frontend can convert data from code units into `cgs` properly, based on the unit definitions from metadata.
```

Comment:
I don't have a personal preference between singular and plural form for "data", but we should be consistent about which is used. You used "the metadata is" earlier, and "data are" here.

Reply:
Ah let's use plural form!